emergence_lib/src/hive_mind.rs

```diff
@@ -29,28 +33,57 @@ pub enum HiveMindAction {
     PlaceAttractivePheromone,
     /// Place a repulsive pheromone.
     PlaceRepulsivePheromone,
+    /// Toggle tilemap labels tools
+    ShowDebugLabels,
+    /// Toggle rendering info
+    ShowInfoText,
+}
+
+// This needs to be converted to LWIM from just inputs
+/// Interface for player controls
+#[derive(Resource)]
+pub struct HiveMindControls {
```

Comment:
I am not convinced this needs to be a separate struct, it's only use is to provide named constants, right? 

Not yet sure, but I think this should be differently somehow

Reply:
There is a better way andI tried to convey that with "This needs to be converted to LWIM from just inputs." I'll change it to "Rework to better use LWIM"